saleor/checkout/utils.py

```diff
@@ -727,7 +747,7 @@ def add_voucher_to_checkout(
         lines,
         address,
     )
-    checkout.voucher_code = voucher.code
+    checkout.voucher_code = voucher_code.code
```

Comment:
This line should be removes. The `voucher_code` is assigned again in 770

Reply:
This one is assigning  string with 'code` to `Checkout` model, line 770 is assigning `VoucherCode` instance to `CheckoutInfo` object